api/tests/integration/middleware/test_cache_control.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import reverse
+
+
+def test_response_have_cache_control_headers(project, admin_user, admin_client):
```

Comment:
Couldn't we just unit test the middleware here rather than having to make a request?

Reply:
But we need to test that the middleware is installed as well, right?

Reply:
I would say not, no. It's kind of config, right - You wouldn't test an ini file for example. It feels kind of analogous to that to me.